src/io-methods/default.js

```diff
@@ -29,11 +29,15 @@ exports.readManifest = function(env) {
 
 exports.writeManifest = function(data, env) {
   var filePath = getFilePath(env)
-  if ( _.startsWith(filePath, 's3://') ) {
+  if (useS3(filePath)) {
     //use s3
     return s3.writeManifest(filePath, data)
   } else {
     //use local file
     return fs.writeManifest(filePath, data)
   }
 }
+
+function useS3(filePath) {
```

Comment:
pretty nit picky: the name doesn't _exactly_ describe what this function does haha. more like `useSupportService` or something. ugh. 

you can leave it though, it's not major.

Reply:
I think `useS3` is more clear -- Digital Ocean is using the AWS S3 sdk to communicate with DO Spaces. Hypothetical future IO Methods could also be support services but would not use the same s3.js file.